<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageCTLWgtClass</class>
 <widget class="QWidget" name="PageCTLWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Control mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="controlModeCombo">
     <item>
      <property name="text">
       <string>Control disabled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Joystick</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Buttons</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Left button</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Pushed position</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="leftBtnPushed">
        <item>
         <property name="text">
          <string>close</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>open</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Right button</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Pushed position</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="rightBtnPushed">
        <item>
         <property name="text">
          <string>close</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>open</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Speed settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Timeout</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Joystick</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>High end</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Center</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Low end</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="joyHighEndSpin">
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="joyCenterSpin">
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="joyLowEndSpin">
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Exp factor</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="expFactorSpin">
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Dead zone</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="deadZoneSpin">
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="reverseJoyChk">
        <property name="text">
         <string>Reverse joystick</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QPushButton" name="joyCalibBtn">
        <property name="text">
         <string>Joystick calibration</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Click settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Max click time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="maxClickTimeSpin">
        <property name="suffix">
         <string> ms</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Delta</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="StepSpinBox" name="deltaPositionBox" native="true">
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StepSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>stepspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>controlModeCombo</tabstop>
  <tabstop>joyLowEndSpin</tabstop>
  <tabstop>joyCenterSpin</tabstop>
  <tabstop>joyHighEndSpin</tabstop>
  <tabstop>expFactorSpin</tabstop>
  <tabstop>deadZoneSpin</tabstop>
  <tabstop>joyCalibBtn</tabstop>
  <tabstop>leftBtnPushed</tabstop>
  <tabstop>rightBtnPushed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
